util/pace/util/grid/stretch_transformation.py

```diff
@@ -22,66 +27,73 @@ def direct_transform(
     centeres on lon_target, lat_target.
 
     Args:
-        lon (in) in radians
-        lat (in) in radians
-        stretch_factor (in) stretch_factor (e.g. 3.0 means that the resolution on
-            tile 6 becomes 3 times as fine)
+        lon (in) in radians # Quantity or ndarray (cupy or numpy)
+        lat (in) in radians # Quantity or ndarray (cupy or numpy)
```

Comment:
This is already stated in the type hint. Can you remove it below too?

```suggestion
        lon (in) in radians
        lat (in) in radians
```